chrometool/chrometool.go

```diff
@@ -138,3 +153,146 @@ func EvalNodeCenter(box *dom.BoxModel) (float64, float64, error) {
 
 	return x, y, nil
 }
+
+func SelectNthElement(ctx context.Context, sel NthChildSel) error {
+	var res interface{}
+	if err := chromedp.Evaluate(fmt.Sprintf(selectNthElementJS, sel.Selector, sel.Position), &res).Do(ctx); err != nil {
+		if err.Error() == "encountered an undefined value" {
+			err = nil
+		}
+		return err
+	}
+	return nil
+}
+
+func ClipboardCommand(ctx context.Context, command string) error {
+	var res interface{}
+	if err := chromedp.Evaluate(fmt.Sprintf(clipboardInteractJS, command), &res).Do(ctx); err != nil {
+		return err
+	}
+	return nil
+}
+
+// DispatchKeyEventToBrowser takes up the key's value to be dispatched to the browser usually the result of keybinding.InterpretKeycode
+// It clicks on the element if isFocussed is false and then sends the appropiate key with the modifiers
+// The modifiers here refer to the xproto modifiers not to be confused with the ones in input.DispatchKeyEvent
+func DispatchKeyEventToBrowser(ctx context.Context, sel NthChildSel, str string, modifiers uint16, isFocussed bool) error {
+	if isFocussed {
+		if err := chromedp.Run(ctx,
+			ClickNthElement(sel, chromedp.ByQueryAll),
+		); err != nil {
```

Comment:
nit: Go doesn't have line length constraints. This would read better in a single line